12/umbraco-cms/tutorials/editors-manual/getting-started-with-umbraco/creating-saving-and-publishing-content.md

```diff
@@ -51,7 +51,7 @@ To set up scheduled publishing, follow these steps:
 3. Select **Schedule**.
 4.  In the **Scheduled Publishing** window, set the date and time in the **Publish at** field.
 
-    ![Schedualed publishing.](../../../../../10/umbraco-cms/tutorials/editors-manual/getting-started-with-umbraco/images/Schedule\_publishing\_v9.png)
+    ![Scheduled publishing.](../../../../../10/umbraco-cms/tutorials/editors-manual/getting-started-with-umbraco/images/Schedule\_publishing\_v9.png)
```

Comment:
to keep the docs consistent, can you please make this change to v10 and v11 as well?